lab_share_lib/processing/rabbit_message_processor.py

```diff
@@ -33,10 +45,21 @@ def _build_processor_for_subject(self, subject: str) -> BaseProcessor:
             BaseProcessor, processor_instance_builder(self._schema_registry, self._basic_publisher, self._config)
         )
 
+    def build_avro_encoder(self, encoder_type, subject):
+        if encoder_type not in ENCODERS.keys():
+            raise Exception(f"Encoder type {encoder_type} not defined")
```

Comment:
Small thing, but would it be clearer to say `not recognised` instead of `not defined` ? Because this is when there is one present, but it's not in the list of recognised ones?

Reply:
Ok